modules/rostests/apitests/comctl32/imagelist.c

```diff
@@ -0,0 +1,138 @@
+/*
+ * PROJECT:         ReactOS api tests
+ * LICENSE:         GPL - See COPYING in the top level directory
+ * PURPOSE:         Test for imagelist
+ * PROGRAMMERS:     Whindmar Saksit
+ */
```

Comment:
Could you use the [new headers](https://reactos.org/wiki/Coding_Style#File_Structure) [formatting](https://git.reactos.org/?p=reactos.git;a=blob;f=modules/rostests/apitests/mountmgr/QueryDosVolumePaths.c;h=e7103b121e7c978f7a1d19c697344fdcb40b0631;hb=97ea95c80cee668dd3a1141146b90eae182483e4#l1)?